src/detect-engine-hsbd.c

```diff
@@ -3631,6 +3709,1227 @@ static int DetectEngineHttpServerBodyFileDataTest03(void)
     return result;
 }
 
+/*
+ * Tests buffer inspection like in ips mode,
+ * here all chunks size are smaller than the window_size.
+
+ * in:               buffer:
+ * [ab]              [ab]
+ * [cd]              [abcd]
+ * [ef]              [cdef]
+*/
+static int DetectEngineHttpServerBodyFileDataTest04(void)
+{
+
+    char input[] = "\
+%YAML 1.1\n\
+---\n\
+libhtp:\n\
+\n\
+  default-config:\n\
+\n\
+    http-body-inline: yes\n\
+    response-body-minimal-inspect-size: 1\n\
```

Comment:
these tests are odd: response-body-minimal-inspect-size will normally be quite a bit bigger than response-body-inspect-window, but all tests use the opposite.